src/workerd/api/tests/compat-flags-test.js

```diff
@@ -0,0 +1,18 @@
+import {
+  ok,
+  throws,
+} from 'node:assert';
+
+import { compatFlags } from 'cloudflare:workers';
+
+export const compatFlagsTest = {
+  test() {
+    throws(() => compatFlags.no_nodejs_compat_v2 = true);
+    throws(() => compatFlags.not_a_real_compat_flag = true);
+    ok(compatFlags['nodejs_compat_v2']);
+    ok(!compatFlags['no_nodejs_compat_v2']);
+    ok(compatFlags['url_standard']);
+    ok(!compatFlags['url_original']);
+    ok(!compatFlags['not-a-real-compat-flag']);
```

Comment:
```suggestion
    ok(!compatFlags['not-a-real-compat-flag']);
    const entries = Object.entries(compatFlags);
    ok(entries.length > 0);
    ok(entries.find(([key,]) => key === 'nodejs_compat_v2')[1]);
    ok(!entries.find(([key,]) => key === 'no_nodejs_compat_v2')[1]);
```
or something along those lines, so that we can also test that the entries are enumerable? 🙂
(on the top of my head I can find of a great use case example for this, but I find the enumerability of the flags to be a nice to have feature 😄)

Reply:
Easiest would be just to call `Object.keys(compatFlags)` and check for specific keys in the resulting array.

Reply:
yes, that would work too 🙂 (since you're already checking the values on the lines above anyways)

Reply:
Done